connect/src/types.ts

```diff
@@ -68,6 +69,18 @@ export interface RedeemedTransferReceipt<AT, SC extends Chain = Chain, DC extend
   destinationTxs?: TransactionId<DC>[];
 }
 
+export interface DestinationQueuedTransferReceipt<
+  AT,
+  SC extends Chain = Chain,
+  DC extends Chain = Chain,
+> extends BaseTransferReceipt<SC, DC> {
+  state: TransferState.DestinationQueued;
+  originTxs: TransactionId<SC>[];
+  attestation: Required<AT>;
+  destinationTxs?: TransactionId<DC>[];
+  queueExpirationTime: Date;
```

Comment:
"expiration" might be a confusing term here... it imply the transfer is no longer valid or eligible. what do you think about `queueReleaseTime`? it's the time the transfer will be released from the queue.

Reply:
Changed